### PR TITLE
[CI:DOCS]  Add missing man page links for Docker man pages 

### PR DESCRIPTION
**Pull request by rhatdan**
Also generate all docker man pages to match podman man pages including links.

#### Does this PR introduce a user-facing change?

```release-note
Adjusted list of Docker equivalent man pages.
```

### CODE REVIEW

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL
@lsm5 @TomSweeneyRedHat PTAL

File: docs/dckrman.sh [hunk 1]
```diff
@@ -1,6 +1,37 @@
...
+	   [[ "$b" =~ "podman-network-update" ]] || \
+	   [[ "$b" =~ "podman-pod" ]] || \
+	   [[ "$b" =~ "podman-system-connection" ]] || \
+	   [[ "$b" =~ "podman-system-migrate" ]] || \
+	   [[ "$b" =~ "podman-system-renumber" ]] || \
+	   [[ "$b" =~ "podman-volume-export" ]] || \
+	   [[ "$b" =~ "podman-volume-import" ]] || \
+	   [[ "$b" =~ "reload" ]] || \
+	   [[ "$b" =~ "unmount" ]] || \
+	   [[ "$b" =~ "unshate" ]]; then
```
Comment (edsantiago) [inline docs/dckrman.sh:32]: typo, I think: `s/ate/are/`.

I'm uncomfortable with all the duplicated syntactic-sugar and with the hardcoding, but can't come up with anything better right now.

**Comment by Luap99** [comment]
Is there a specific reason to exclude the podman only pages? I see no problem with including more docs. If you run in compat mode and you do `docker --help` it will show you all commands so as user I would expect to have the man pages for them as well.

Also I don't really think it is maintainable, nobody is going to remember to add new podman commands here when one is added. Sure this is not a big deal but then why bother in the first place.

**Comment by vrothberg** [comment]
I also share @Luap99's concerns.  If I do a `--help`, I will see the commands but there's no man page to it.

**Comment by rhatdan** [comment]
@edsantiago @Luap99 @vrothberg Dropped the dropping of non docker man page equivalents.

But now include the additional linked man pages.

man docker-create 

Currently exists

But 
man docker-container-create did not,

This PR fixes this.

**Pull request merged**